src/api/utils/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,33 @@
+import { useContext } from 'react';
+import { TodosContext } from '../TodoContext';
+import { TodoTitleField } from '../TodoTitleField';
+
+export const TodoList:React.FC = () => {
+  const {
+    filteredTodos,
+    tempTodo,
+  } = useContext(TodosContext);
+
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {filteredTodos.length !== 0
+      && (
```

Comment:
Fix all cases
```suggestion
      {filteredTodos.length !== 0 && (
```

Reply:
not fixed
<img width="621" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/98764641/4dfba427-35ab-4d7c-9230-cc2108c49527">